lpcompiler.pas

```diff
@@ -2760,9 +2845,11 @@ function TLapeCompiler.ParseExpression(ReturnOn: EParserTokenSet = []; FirstNext
           end;
         {$ENDIF}
 
-        ParserToken_FirstOperator..ParserToken_LastOperator: ParseOperator();
         else
-          Break;
+          if Tokenizer.Tok in [ParserToken_FirstOperator..ParserToken_LastOperator] then
```

Comment:
`tk_sym_BracketOpen` is in `ParserToken_FirstOperator..ParserToken_LastOperator`
so the case statement wont compile if we keep it as is, with `ParserToken_FirstOperator..ParserToken_LastOperator` being one of the possible cases. So that case had to be moved to the `else`-statement.